catalog/rest/catalog-rest-endpoint/src/main/java/org/codice/ddf/endpoints/rest/action/AbstractMetacardActionProvider.java

```diff
@@ -105,7 +108,14 @@ public String getId() {
 
     @Override
     public <T> boolean canHandle(T subject) {
-        return subject instanceof Metacard;
+        if (subject instanceof Metacard) {
```

Comment:
I'd argue the opposite. Having a variable to keep track of while reading code makes it more difficult to understand than explicit returns. explicit > implicit, imo.